LEAF_Request_Portal/api/controllers/EmailTemplateController.php

```diff
@@ -8,80 +8,66 @@
 class EmailTemplateController extends RESTfulResponse
 {
     public $index = array();
-
     private $API_VERSION = 1;    // Integer
-
     private $emailTemplate;
-
     private $db;
-
     private $login;
-
     public function __construct($db, $login)
     {
         $this->db = $db;
         $this->login = $login;
         $this->emailTemplate = new EmailTemplate($db, $login);
     }
-
     public function get($act)
     {
         $db = $this->db;
         $login = $this->login;
         $emailTemplate = $this->emailTemplate;
-
         $this->index['GET'] = new ControllerMap();
-
         $this->index['GET']->register('emailTemplates', function ($args) use ($emailTemplate) {
             return $emailTemplate->getEmailAndSubjectTemplateList();
         });
-
-        $this->index['GET']->register('emailTemplates/[text]', function($args) use ($emailTemplate) {
+        $this->index['GET']->register('emailTemplates/[text]', function ($args) use ($emailTemplate) {
             return $emailTemplate->getEmailTemplate($args[0]);
         });
 
+        $this->index['GET']->register('getEmailTemplateFileHistory/[text]', function ($args) use ($emailTemplate) {
+            return $emailTemplate->getEmailTemplateFileHistory($args[0]);
+        });
+
         $this->index['GET']->register('emailTemplates/[text]/standard', function ($args) use ($emailTemplate) {
             return $emailTemplate->getEmailTemplate($args[0], true);
         });
 
         return $this->index['GET']->runControl($act['key'], $act['args']);
     }
-
     public function post($act)
     {
+        $db = $this->db;
+        $login = $this->login;
         $emailTemplate = $this->emailTemplate;
+        $this->verifyAdminReferrer();
+        $this->index['POST'] = new ControllerMap();
+        $this->index['POST']->register('emailTemplates/[text]', function ($args) use ($emailTemplate) {
+            return $emailTemplate->setEmailTemplate($args[0]);
+        });
+        $this->index['POST']->register('emailTemplateFileHistory/[text]', function ($args) use ($emailTemplate) {
+            return $emailTemplate->setHistoryEmailTemplate($args[0]);
+        });
 
-        $verified = $this->verifyAdminReferrer();
-
-        if ($verified) {
-            echo $verified;
-        } else {
-            $this->index['POST'] = new ControllerMap();
-
-            $this->index['POST']->register('emailTemplates/[text]', function ($args) use ($emailTemplate) {
-                return $emailTemplate->setEmailTemplate($args[0]);
-            });
-
-            return $this->index['POST']->runControl($act['key'], $act['args']);
-        }

```

Comment:
Is there a reason for undoing the RESTfulResponse changes that were released in the last push to prod? This verified check needs to be here in order to echo the response if needed.